crates/rari-doc/src/translations.rs

```diff
@@ -60,7 +62,7 @@ pub(crate) fn init_translations_from_static_docs() {
 /// * `Vec<(Locale, String)>` - Returns a vector of tuples, where each tuple contains a `Locale` and a `String`
 ///   representing the title of the translation. If no translations are found, an empty vector is returned.
 pub(crate) fn get_other_translations_for(slug: &str, locale: Locale) -> Vec<(Locale, String)> {
-    if cache_content() {
+    if cache_content() && slug.contains("/docs/") {
```

Comment:
Afaik `TRANSLATIONS_BY_SLUG` only contains docs pages.

But maybe there is a more elegant way to do this?